gnusocial/README.md

```diff
@@ -0,0 +1,11 @@
+## GNU Social
+
+This Caddyfile should work with any branch of [GNU Social][gs]. For prerequisites and installation information, see the included README file.
+
+To get the latest version, enter the following where you want your [gs][GNU Social] installation located:
+    git clone https://git.gnu.io/gnu/gnu-social.git
+If you are feeling adventurous, you can obtain the bleeding edge version with:
+    git checkout nightly
+while in your gnusocial directory.
```

Comment:
If you could fix the code block formatting (need empty lines on both sides) that would be good :)